python/src/GaussianLinearCalibration_doc.i.in

```diff
@@ -31,9 +31,8 @@ gradientObservations : 2-d sequence of float
 
 Notes
 -----
-GaussianLinearCalibration is known as the *Best Linear Unbiased Estimator* or *BLUE*. 
-It is the minimum variance estimator of the parameter of a given model under 
-the assumption that this parameter acts linearly in the model.
+GaussianLinearCalibration implements the "Maximum A Posteriori" (MAP) estimator for a linear model
```

Comment:
you cannot use double quotes inside, the docstring is already quoted

Reply:
Ok. Fixed.